lib/diagnostic.rb

```diff
@@ -6,21 +6,23 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = 'print or puts'
```

Comment:
We want to run a script from the terminal.  These are Ruby methods that print to the console.  `ruby lib/example.rb`.